python/pyspark/ml/tests.py

```diff
@@ -655,6 +656,20 @@ def test_nested_pipeline_persistence(self):
             except OSError:
                 pass
 
+    def test_write_property(self):
+        lr = LinearRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
+        lr.write.save(lr_path)
```

Comment:
How about just testing:
```
self.assertTrue(isinstance(lr.write, MLWriter))
```
There's no need to actually create a temp directory or save anything in this test.